src/components/structures/SpaceHierarchy.tsx

```diff
@@ -186,9 +187,12 @@ const Tile: React.FC<ITileProps> = ({
         description += " · " + _t("%(count)s rooms", { count: numChildRooms });
     }
 
-    const topic = joinedRoom?.currentState?.getStateEvents(EventType.RoomTopic, "")?.getContent()?.topic || room.topic;
-    if (topic) {
-        description += " · " + topic;
+    let topic;
```

Comment:
Please give this a type

Reply:
This pattern was used throughout the file so I added type annotations to all the other places, too.